R/zzz.R

```diff
@@ -11,6 +11,10 @@
     options("bbr.bbi_exe_path" = BBI_DEFAULT_PATH)
   }
 
+  if (is.null(getOption("bbr.bbi_exe_mode"))) {
+    options("bbr.bbi_exe_mode" = BBI_DEFAULT_MODE)
```

Comment:
when is `zzz.R` called? is this a safe assumption to make?

Reply:
it's called when you load the package (after the rest of the files in `R` are sourced). That's where we do the rest of these so I think it's right. It's basically "if this isn't already set (in `.Rprofile` or something) then set it to the default." Then you can overwrite that in your code later too.

Reply:
okay that seems fine, just wanted to verify